src/main/scala/cromwell/binding/types/WdlObjectType.scala

```diff
@@ -2,4 +2,6 @@ package cromwell.binding.types
 
 case object WdlObjectType extends WdlType {
   override def toWdlString: String = "Object"
+
+  override protected def coercion = ???
```

Comment:
That's fine, I had no idea what to put here; the `???` captured my thinking nicely. :smile: